t/dedupe.c

```diff
@@ -473,9 +535,33 @@ static void show_chunk(struct chunk *c)
 	}
 }
 
-static void show_stat(uint64_t nextents, uint64_t nchunks, uint64_t ndupextents)
+static const char *capacity_unit[] = {"b","KB", "MB", "GB", "TB", "PB", "EB"};
+
+static void bytes_to_human_readable_unit(uint64_t n, uint64_t *n_out, const char **unit_out)
+{
+    uint8_t i;
+
+    *n_out = n;
+    i = 0;
+
+    if (*n_out < 1024) {
+        goto out;
+    }
+
+    while (*n_out >= 1024) {
+        i++;
+        *n_out /= 1024;
+    }
+
+out:
+    *unit_out = capacity_unit[i];
+}
```

Comment:
Use tabs. Also single statement inside "if" doesn't have to be closed with braces.

```suggestion
static void bytes_to_human_readable_unit(uint64_t n, uint64_t *n_out, const char **unit_out)
{
	uint8_t i;

	*n_out = n;
	i = 0;

	if (*n_out < 1024)
		goto out;

	while (*n_out >= 1024) {
		i++;
		*n_out /= 1024;
	}

out:
	*unit_out = capacity_unit[i];
}
```

Reply:
fixed